Pipfile

```diff
@@ -23,4 +23,4 @@ typing_extensions = "*"
 [packages]
 
 [requires]
-python_version = "3.6"
+python_version = "3.8"
```

Comment:
You need to run the command `make init` to update the `Pipfile.lock` and check that file in.

Other than that, the PR looks great and I can merge it.

Reply:
Done. But it leaded to some issues...